lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -3148,6 +3148,48 @@ static void fwdTopK(Tensor *outW, Tensor *indW, Tensor *inW, size_t k) {
   }
 }
 
+template <typename T>
+static void fwdArgMax(Tensor *argmaxW, Tensor *inW, size_t axis) {
+  auto argmaxH = argmaxW->getHandle<int64_t>();
+  auto inH = inW->getHandle<T>();
+
+  auto idim = inW->dims();
+
+  size_t a, b, c, d = 0;
+
+  size_t *dim[4];
```

Comment:
is it strictly necessary to have 4 dims?

Reply:
Curious how it differs from other operators that use ShapeNHWC, which again ties to the 4D data. Now, I understand I could make a generic ArgMax, but that would be an additional effort (plug tests to cover it). Is it ok to have an assert for !4 dims ?

Reply:
we need to make sure that Node's verification handles this properly than.